client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -405,6 +409,8 @@ export class CheckoutThankYou extends Component {
 				purchases.some(
 					( purchase ) => isDomainMapping( purchase ) || isDomainTransfer( purchase )
 				);
+			wasDomainMappingOrRegistration =
+				purchases.some( isDomainRegistration ) || purchases.some( isDomainMapping );
```

Comment:
Two notes here:
1. As noted above RE variable naming it might be simpler to use one `wasDomainProduct` variable.
2. Might it not be better to use a single `.some()` call that applies the `||` within the `.some()` function? We do this in the logic above?
```suggestion
				purchases.some(
					( purchase ) => isDomainMapping( purchase ) || isDomainTransfer( purchase ) || isDomainRegistration( purchase )
				);
```